jinja2/base.html

```diff
@@ -59,7 +59,15 @@
   <!-- non-Retina iPhone, iPod Touch, and Android 2.1+ devices: -->
   <link rel="apple-touch-icon-precomposed" href="{{ static('img/favicon57.png') }}">
   <!-- basic favicon -->
-  <link rel="shortcut icon" href="{{ static('img/favicon32.png') }}">
+     <!-- basic favicon -->
+  {% if settings.DOMAIN == settings.STAGING_DOMAIN %}
+    {% set favicon_suffix = '-staging' %}
+  {% elif settings.DOMAIN == 'localhost' %}
+    {% set favicon_suffix = '-local' %}
+  {% else %}
+    {% set favicon_suffix = '' %}
```

Comment:
The specific cases are production and staging, the fallback should be the `-local` - see the previous version:

https://github.com/mozilla/kuma/blob/14521cb18ad486575b8aeb1459ecb03a3ff6e9dc/jinja2/base.html#L64-L71

Reply:
@jwhitlock In current configuration, there are nothing specific for production. But there are specific case for `dev` and `stage`. So I thought to have specific case for `dev` and `stage` and fallback to `production`

Reply:
The point of the different favicon.ico is to have one that only appears in production, one that only appears in staging, and everywhere else gets the dev icon. It's not OK for the production icon to be the default.

``settings.PRODUCTION_DOMAIN`` should probably be added, and ``PRODUCTION_URL`` should be built from that rather than ``PROTOCOL`` and ``DOMAIN``. Research is needed to see why it was done this way.

Reply:
Agreed! 👍